### DISCUSSION TITLE
Docker Noob: Where are my config files?

### DISCUSSION THREAD

**Discussion by oguruma1218** [dani-garcia/vaultwarden#1939] [General]
I installed this via docker with the help of [ this guide. ](https://theselfhostingblog.com/posts/how-to-self-host-bitwarden-on-ubuntu-server/)

Where exactly are my config files? I set up a /bitwarden directory, but all that's in it are the sqllite file and key files.

Specifically, I need to set up SMTP....

**Comment by Masgalor** [comment]
Vaultwarden can be fully configured by environment variables, so pay attention to the `environment:` section in your compose file.

A config-file will only be created if you change some setiings on the admin page.
See https://github.com/dani-garcia/vaultwarden/wiki/Configuration-overview

**Reply by oguruma1218** [reply]
I've changed my compose file to below, and re-ran `sudo docker-compose up -d`

I still don't see an "Admin" page, or a config file...

`version: '3'

services:
  bitwarden:
    image: bitwardenrs/server
    restart: always
    ports:
      - 8000:80
    volumes:
      - ./bw-data:/data
    environment:
      WEBSOCKET_ENABLED: 'true' # Required to use websockets
      SIGNUPS_ALLOWED: 'true'   # set to false to disable signups
      ADMIN_TOKEN: 'myadmintoken'`

**Reply by RealOrangeOne** [reply]
You won't see a config file, all config is done through environment variables. Why are you expecting a config file?

When you say "You don't see an admin page", where are you looking?
